src/App.jsx

```diff
@@ -14,70 +15,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const click = (good) => {
+    setSelectedGood(good);
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {
+        !selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              No goods selected
+            </h1>
+          )
+          : (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => click('')}
+              />
+            </h1>
```

Comment:
you put button in `h1`? what?!

Reply:
This is Mate academy code:

<h1 className="title is-flex is-align-items-center">
      Jam is selected

      <button
        data-cy="ClearButton"
        type="button"
        className="delete ml-3"
      />
    </h1>